views/js/pciCreator/dev/audioRecordingInteraction/runtime/js/recorder.js

```diff
@@ -200,8 +199,35 @@ define([
                         initAnalyser(stream);
 
                         setState(recorder, recorderStates.IDLE);
+                    })
+                    .catch(function(err) {
+                        alert(err);
                     });
             },
+
+            /**
+             * Create the Audio context
+             * @returns {AudioContext}
+             */
+            initAudioContext: function initAudioContext() {
+                var context = window.audioContext;
+
+                if (context) {
+                    audioContext = context;
+                } else {
+                    var AudioContext = window.AudioContext || window.webkitAudioContext;
+                    audioContext = new AudioContext();
+                }
+
+                if (audioContext && (audioContext.state !== 'running')) {
+                    audioContext.resume();
```

Comment:
`resume()` returns a Promise and currently the error is not handled. It gets rejected if the AudioContext is already been closed.